src/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Activity.cs

```diff
@@ -343,7 +350,88 @@ public void Stop()
             }
         }
 
+        /* W3C support functionality (see https://w3c.github.io/trace-context) */
+
+        /// <summary>
+        /// Holds the W3C 'tracestate' header.   This is typically used by logging systems
+        /// to store vendor-specific information that must flow.
+        /// Logically it is just a kind of baggage (if flows just like baggage), but because
+        /// it is expected to be special cased (it has its own HTTP header), it is more 
+        /// convenient/efficient if it is not lumped in with other baggage.   
```

Comment:
I don't think this summary is helpful for the person uses this API. I'd suggest the following text:

> The filed `tracestate` carries information supplemental to trace identity contained in `traceparent`. List of key value pairs carries by `tracestate` convey information about request position in multiple distributed tracing graphs. It is typically used by distributed tracing systems and should not be used as a general purpose baggage as this use may break correlation of a distributed trace.

Reply:
I will add this to the comment